README.md

```diff
@@ -6,7 +6,7 @@ rpmoci builds OCI container images from RPM packages, using [DNF](https://github
 rpmoci features:
 
  - **deterministic** rpmoci locks RPM dependencies using the package file/lockfile paradigm of bundler/cargo etc and supports vendoring of RPMs for later rebuilds.
- - **no container runtime required** rpmoci can build images in environments without docker access.
+ - **unprivileged** rpmoci can build images in environments without docker access, and without root access (this relies on the user being able to create [user namespaces](https://www.man7.org/linux/man-pages/man7/user_namespaces.7.html))
```

Comment:
You'll need more docs here for the pre-requisites if you take some of my advice below (setting up subuids through `/etc/subuid` or `nss`, installing `newuidmap` and `newgidmap`...)

Weirdly I think rootless mode can't work _in_ a container, since docker bans the use of `unshare` (and `clone3`) in containers (without `--privileged`)....

Reply:
yep, I've added more docs and pointed to the rootless container docs which act as a good signpost for this space